roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+from openshift_checks.mixins import NotContainerizedMixin
+
+
+class DockerImageAvailability(NotContainerizedMixin, OpenShiftCheck):
+    """Check that required Docker images are available."""
+
+    name = "docker_image_availability"
+    tags = ["preflight"]
+
+    def run(self, tmp, task_vars):
+        openshift_release = get_var(task_vars, "openshift_release")
+
+        # attempt to fetch fully qualified image tag
+        openshift_facts = self.module_executor("openshift_facts", {}, tmp, task_vars)
+        openshift_image_tag = self.get_image_tag(openshift_facts)
+
+        failed_pulls = set()
+
+        # attempt to pull each image and fail on
+        # the first one that cannot be fetched
```

Comment:
This looks like function/method docstring. It would be best to put it on the method definition, not on its call site.